build.gradle

```diff
@@ -287,10 +287,11 @@ dependencies {
 
     compile group: 'com.sun.xml.bind', name: 'jaxb-osgi', version: '2.3.3'
 
-    compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
-    compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
-    compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.reformS2sClient
+    implementation 'com.github.hmcts.java-logging:logging:5.1.7'
```

Comment:
can you upgrade versions.reformLogging to latest and use that instead of using hard code `5.1.7`